### DISCUSSION TITLE
Hello All - Issue on image upload on subdomain install, path issue, any help appreciated.

### DISCUSSION THREAD

**Discussion by CarlisleCo** [givanz/Vvveb#171] [General]
I see in the demo when you add an image somewhere, then set the image, the path is ex: /vvvebjs/media/1.jpg

I have installed this on builder.mysite.com, shared hosting & cPanel, everything works but when I add an image and try to set the image I see the path is: https://builder.mysite.com/public/js/vvvebjs/icons/image.svg. which obviously will not work.

Any idea on how to change path, I looked at the htaccess, I am really not sure on that one, & set the Site domain/subdomain settings as builder.mysite.com.

Would be grateful for any input, thank you kindly!
- Daniel

**Comment by givanz** [comment]
Where do you set the image? 

If you upload the image from media https://demo.vvveb.com/admin/index.php?module=media/media the the image will be saved to `/public/media` https://demo.vvveb.com/media/image.png

Images from `public/js/vvvebjs/icons/` folder are used for VvvebJs components https://github.com/givanz/Vvveb/blob/master/public/js/vvvebjs/components-html.js#L117

**Comment by CarlisleCo** [comment]
Thank you for the reply. I am not sure what what going really. I was getting errors on upload and when it did upload an image it would not let me set the image. I installed it again and same but let me upload images... After the 3rd fresh install, it seem to be working, crazy!
I really do appreciate your reply and I look forward to seeing how this works, great job my friend. 
- Daniel
